connect/api/src/main/java/org/apache/kafka/connect/sink/SinkTaskContext.java

```diff
@@ -25,6 +25,12 @@
  * Context passed to SinkTasks, allowing them to access utilities in the Kafka Connect runtime.
  */
 public interface SinkTaskContext {
+
+    /**
+     * Get the Task configuration.
```

Comment:
Need more JavaDoc to define the behavior. Does this return null? Will the result ever differ if called multiple times? How does this relate to the configuration passed to the connector and/or task upon startup?